### DISCUSSION TITLE
What is the difference between component/#translation 'manage strings' and component/#files  'edit base file'?

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#10154] [Q&A] [answered]
When configuring a component in hosted.weblate.org, there are two options that I disable, and I wonder, are they the same? 

When I forget to disable them, I get conflicts between the development repository and weblate.

**Comment by nijel** [comment, marked as answer]
See https://docs.weblate.org/en/latest/admin/projects.html#manage-strings and https://docs.weblate.org/en/latest/admin/projects.html#edit-base-file. I've tried to further clarify this in https://github.com/WeblateOrg/weblate/pull/10157
